schema/asset_uploads/create_asset_request_mutation.js

```diff
@@ -93,6 +93,6 @@ export default mutationWithClientMutationId({
   },
   mutateAndGetPayload: ({ name, acl }, request, { rootValue: { createNewGeminiAssetLoader } }) => {
     if (!createNewGeminiAssetLoader) return null
-    return createNewGeminiAssetLoader({ name, acl })
+    return createNewGeminiAssetLoader(name, acl)
```

Comment:
IMO named parameters are always better than the other way around as there are no ordering restrictions and intellisense is better. Why did you remove?

Reply:
Hmm, that's true. I removed them because they are required, but if the convention is to prefer objects, I'll conform to that 👍

Reply:
I don't think its a convention but with ES6's object shorthand syntax its much more common these days